.github/workflows/cibuildwheel.yml

```diff
@@ -126,7 +160,7 @@ jobs:
         vcpkg install openssl:x86-windows
```

Comment:
I think the condition for this job needs to be updated to also cover `CIBW_BUILD` ending with "win_amd64"

Reply:
or maybe instead look at whether `os` starts with "windows-"

Reply:
The `win_amd64` build had no issues previously with `windows-2022`...I'll think I'll just revert to that as the `windows-2025` image is still classed as `beta` & there are still tools etc. been added/updated. (will look in to using it again in a few weeks when v3 of cibuildweel action is released.)

Is that ok with you??

Reply:
sure

Reply:
I'll rebase/make changes in about an hour or so.

Reply:
Thoughts on sync? -> https://github.com/arvidn/libtorrent/pull/7805#issuecomment-2613052694

Reply:
https://github.com/arvidn/libtorrent/pull/7824